### ISSUE TITLE
should use common config

### ISSUE THREAD

**Issue by c-e-p-x-u-o** [prodrigestivill/roundcube-gravatar#3]
It should use 
    $this->load_config();
without parameters,

not
    $this->load_config('config.inc.php.dist');
    $this->load_config('config.inc.php');

**Comment by prodrigestivill** [comment]
Current code load default variables first and then the ones provided by the user so that the user doesn't need to define all variables.

_Proposed solution not accepted._

**Comment by c-e-p-x-u-o** [comment]
Current code does not load config.inc.php from /config directory at all.

**Comment by prodrigestivill** [comment]
For installing issues please refer to [README>Installation](https://github.com/prodrigestivill/roundcube-gravatar/blob/master/README.md#Installation).

File `config.inc.php`should be placed in the folder `plugins/gravatar/` of your roundcube installation, also an example can be found as `config.inc.php.dist` in the same folder.

Any proposed change of location for this files will break any previous installations, and also is not how current roundcube plugins manage their configuration.

The current implementation is using the same config file pattern/folder as the rest of plugins, please refer to
[rcube_plugin.php](https://github.com/roundcube/roundcubemail/blob/430c000e323ca2e12981cbf2b9351f960272116c/program/lib/Roundcube/rcube_plugin.php) for more information:
```php
public function load_config($fname = 'config.inc.php') {}
```
- so basically:
```php
$this->load_config('config.inc.php');
```
- is the same as:
```php
$this->load_config();
```
And code just prepopulate first with the default settings from `config.inc.php.dist`.

**Comment by prodrigestivill** [comment]
This is not true. Both calls you put there are exactly the same call for the defaults of the function arguments.

The only change in behavior is that if you don't have config.inc.php in the plugin folder default will overriden the `/config` settings.

_Reopened for future improvement._
